axe_selenium_python/axe.py

```diff
@@ -93,17 +93,19 @@ def report(self, violations):
             string += "\n\n\n"
         return string
 
-    def write_results(self, data, name="results.json"):
+    def write_results(self, data, name):
         """
         Write JSON to file with the specified name.
 
         :param name: Name of file to be written to.
         :param output: JSON object.
         """
-        filepath = os.path.join(os.getcwd(), name)
+        filepath = os.path.abspath(name)
 
         with open(filepath, "w", encoding="utf8") as f:
             try:
                 f.write(unicode(json.dumps(data, indent=4)))
             except NameError:
                 f.write(json.dumps(data, indent=4))
+            except FileNotFoundError as fnf_error:
+                print(fnf_error)
```

Comment:
A better experience would be to create the file if it doesn't exist.

Reply:
This section can be removed. @kimberlythegeek suggested that we write to the current directory if there's no path passed to `write_results`. I intend to do  something like `open(filepath, 'w')` This will make sure that a file is always created, either in the current directory or in the user specified path. I'll submit a new commit now with changes.